src/meta/src/controller/system_param.rs

```diff
@@ -143,7 +143,6 @@ impl SystemParamsController {
         let params = SystemParameter::find().all(&db).await?;
         let params = merge_params(system_params_from_db(params)?, init_params);
 
-        info!("system parameters: {:?}", params);
```

Comment:
We're going to set a key for integration tests in CI. Removing these log lines to prevent any information leaks. 🥵

Reply:
A thorough solution is to not impl `Debug` for the `SystemParams` message with https://docs.rs/prost-build/latest/prost_build/struct.Config.html#method.skip_debug. However it's not possible in `tonic-build` now.

Reply:
As you have redacted the license key in #17936 [here](https://github.com/risingwavelabs/risingwave/pull/17936/files#diff-80501f8404fc9112559001fe6018516af01e7ac530dec28a8c6d4aef489074bdR81-R85), shall we put back the `info!` log?

Reply:
Sure. [`343cdae` (#17936)](https://github.com/risingwavelabs/risingwave/pull/17936/commits/343cdaecdd739415018ac0440a583bfff0c6d421)